src/broadcast/metadatabroadcast.cpp

```diff
@@ -0,0 +1,77 @@
+
+#include <QLinkedListIterator>
+
+#include "broadcast/metadatabroadcast.h"
+#include "mixer/playerinfo.h"
+
+MetadataBroadcaster::MetadataBroadcaster()
+{
+
+}
+
+void MetadataBroadcaster::slotAttemptScrobble(TrackPointer pTrack) {
+    for (auto it = m_trackedTracks.begin();
+         it != m_trackedTracks.end();
+         ++it) {
+        if (*it == GracePeriod(0,pTrack)) {
+            GracePeriod &trackPeriod = *it;
+            if (trackPeriod.hasBeenEjected &&
+                trackPeriod.m_msElapsed > 
```

Comment:
I don't understand what you mean here, which line break? All of them?

Reply:
line brakes because of long lines = 8 spaces. 
Parentheses alignment, does not work, because it is 4 spaces you cannot distinguish it from logical indent.   
```
if (trackPeriod.hasBeenEjected &&
        trackPeriod.m_msElapsed > 
```